processing/src/main/java/io/druid/segment/StorageAdapter.java

```diff
@@ -63,8 +64,6 @@
   @Nullable
   public ColumnCapabilities getColumnCapabilities(String column);
 
-  public Map<String, DimensionHandler> getDimensionHandlers();
```

Comment:
It was unused, and I didn't want it to be part of the public API of StorageAdapter.